code/parse/sexp.cpp

```diff
@@ -34771,6 +34862,14 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"\t2:\tThe default text if no localized version is available.\r\n"
 		"\t3:\tThe XSTR index. If set to -1 then the default value will be used\r\n"
 	},
+
+	{ OP_REPLACE_TEXTURE, "replace-texture\r\n"
+		"\tChanges a texture of a ship to a different texture, similar to the FRED texture replace.\r\n"
+		"Takes 3 arguments...\r\n"
+		"\t1: Name of the Ship.\r\n"
+		"\t2: Name of the texture to be replaced.\r\n"
+		"\t3: Name of the texture to be changed to.\r\n"
+	},
```

Comment:
To enhance usability I suggest changing the arguments as follows:
```
		"Takes 3 or more arguments...\r\n"
		"\t1: Name of the texture to be replaced.\r\n"
		"\t2: Name of the texture to be changed to.\r\n"
		"\tRest: Name of the ship or wing.\r\n"
```
This would be more useful if the FREDder had a number of ships or wings of identical models that all required the same replacement.